etna/packages/etna-js/components/inputs/dropdown_autocomplete.jsx

```diff
@@ -67,17 +68,25 @@ export default function DropdownAutocomplete({
 
   useEffect(() => {
     onChange = debounce(onChange, waitTime || 500);
-    setSelectedValue(defaultValue ? defaultValue : '');
+    setSelectedValue(defaultValue ? defaultValue : null);
```

Comment:
Do we feel comfortable with this change in general?  The other DropdownAutocomplete uses do not depend on this?

Reply:
I don't think they do, but not 100% sure. Can try to verify. I only changed it because of some warning / recommendation from running tests, I think end behavior is the same even if we leave it as `''`.

Reply:
Just tried this against the other Vulcan UI components, and the change seems to be okay / have no impact.

Reply:
reverted this to be safe, since the original didn't seem to result in negative behavior.